src/AVFoundation/AVAssetDownloadStorageManagementPolicy.cs

```diff
@@ -18,7 +20,12 @@ public partial class AVMutableAssetDownloadStorageManagementPolicy {
 		
 		public override AVAssetDownloadedAssetEvictionPriority Priority {
 			get { return AVAssetDownloadedAssetEvictionPriorityExtensions.GetValue (_Priority); }
-			set { _Priority = value.GetConstant (); }
+			set
+			{
+				var val = value.GetConstant ();
+				if (val is not null)
+					_Priority = val;
+			}
```

Comment:
This is a change in behavior (which we don't want to do); in other places we've done this:

```suggestion
			set { _Priority = value.GetConstant ()!; }
```

Reply:
@rolfbjarne  Question about this one, in this example, `_Priority` cannot be null and we are setting it equal to something that could possibly be null. Since we know that the GetConstant was not returning null in the past (because _Priority cannot be null) then it is okay that we tell the compiler we know that this value won't be null?

Reply:
It's ok to tell the compiler we know it won't be null, because that's how it behaved in the past, and we don't want to change the behavior.